Classes/Query/AbstractSearchRequestBuilder.php

```diff
@@ -25,19 +24,18 @@ abstract class AbstractSearchRequestBuilder implements ProtectedContextAwareInte
 
     protected bool $logThisQuery = false;
 
-    protected string $logMessage;
+    protected ?string $logMessage;
     private WorkspaceName $workspaceName;
 
+    /**
+     * @param array<mixed> $additionalAliases
+     */
     public function __construct(
-        protected readonly ContentRepositoryRegistry $contentRepositoryRegistry,
         protected readonly Elasticsearch $elasticsearch,
         protected readonly Node|null $contextNode = null,
         protected readonly array $additionalAliases = [],
-
     ) {
-        $contentRepository = $this->contentRepositoryRegistry->get($this->contextNode->subgraphIdentity->contentRepositoryId);
```

Comment:
it seems that `$contextNode` is never null because the code previously just always accessed `subgraphIdentity`.